### PR TITLE
Commit during guest stop/cont

### PR DESCRIPTION
**Pull request by aliang123**
blockdev_commit_stop_cont:commit during guest stop/cont

Signed-off-by: Aihua Liang <aliang@redhat.com>

id: 1820515

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_stop_cont.q35: PASS (74.92 s)

**Comment by aliang123** [comment]
@ngu-niny, @zhencliu, please help to review.

**Comment by aliang123** [comment]
depends on https://github.com/autotest/tp-qemu/pull/2182

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_stop_cont.q35: PASS (41.66 s)

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.blockdev_commit_stop_cont.q35: PASS

File: qemu/tests/blockdev_commit_stop_cont.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+from provider import backup_utils
+from provider import job_utils
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitStopCont(BlockDevCommitTest):
+
+    def configure_data_disk(self, tag):
+        os_type = self.params["os_type"]
```
[the remark this answers is not in the document]
Comment (aliang123) [reply qemu/tests/blockdev_commit_stop_cont.py]: Have updated it already.

File: qemu/tests/blockdev_commit_stop_cont.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+from virttest import utils_disk
+
+from provider import backup_utils
+from provider import job_utils
+
+from provider.blockdev_commit_base import BlockDevCommitTest
```
[the remark this answers is not in the document]
Comment (aliang123) [reply qemu/tests/blockdev_commit_stop_cont.py:4]: Hi, xutian
   I think import BlockdevCommitTest is better.
   If we can run blockdev_commit_base.py individually, from from provider import blockdev_commit_base will import its __mian__. And in our test, only the class import is enough.

File: qemu/tests/cfg/blockdev_commit_stop_cont.cfg [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    image_format_sn3 = qcow2
+
+    image_name_sn4 = sn4
+    image_format_sn4 = qcow2
+
+    device_tag = "image1"
+    rebase_mode = unsafe
+    mount_point = "/tmp"
+    qemu_force_use_drive_expression = no
+    Windows:
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_commit_stop_cont.cfg:28]: Can we remove it as 'only Linux'?

**Comment by aliang123** [comment]
Has removed "Windows" part in cfg.

**Comment by zhencliu** [comment]
LGTM

**Comment by ngu-niny** [comment]
Test PASS:

'09-repeat7.Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.blockdev_commit_stop_cont' Finshed 2020-04-15 08:32:37 2020-04-15 08:35:30 PASS 172.999561

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by YongxueHong** [comment]
LGTM

**Comment by aliang123** [comment]
@YongxueHong, @zhencliu, @ngu-niny, @xutian, thanks all for your review.

**Pull request merged**